.github/workflows/release-console.yml

```
@@ -0,0 +1,88 @@
+name: Release Console
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseVersion:
+        description: 'Release version (e.g., v1.2.3)'
+        required: true
+      commitHash:
+        description: 'Commit hash to tag'
+        required: true
+      environment:
+        description: 'Deployment environment (main or testnet)'
+        required: true
+        default: 'main'
+
+env:
+  E2E_TEST_WORKFLOW: ./.github/workflows/ci-trading-python-tests.yml
+
+jobs:
+  setup_release_branch:
+    runs-on: ubuntu-latest
+    outputs:
+      branch_name: ${{ steps.prepare_branch.outputs.branch_name }}
+    steps:
+      - name: Checkout code at commit hash
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.inputs.commitHash }}
+
+      - name: Create Tag
+        run: git tag ${{ github.event.inputs.releaseVersion }} ${{ github.event.inputs.commitHash }}
+
+      - name: Push Tag
+        run: git push origin ${{ github.event.inputs.releaseVersion }}
```

I thin you want this:

```suggestion
        run: git push origin tag ${{ github.event.inputs.releaseVersion }}
```